WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -115,6 +115,7 @@
         NEWS_CARD_SHOWN_VERSION,
         AVATAR_VERSION,
         GUTENBERG_DEFAULT_FOR_NEW_POSTS,
+        USER_IN_GUTENBERG_ROLLOUT_GROUP,
```

Comment:
I think the new pref constant is not used in the code? Should the `isUserInGutenbergRolloutGroup()` and `setUserInGutenbergRolloutGroup()` methods use it?

Reply:
Well spotted 🤦‍♂ - copy/paste failure! Updated in 2ff8e8d